command.go

```diff
@@ -259,13 +261,22 @@ func ExecuteSetLineHeight(c Command, v *VHS) {
 
 // ExecuteSetTheme applies the theme on the vhs.
 func ExecuteSetTheme(c Command, v *VHS) {
-	err := json.Unmarshal([]byte(c.Args), &v.Options.Theme)
-	if err != nil {
-		fmt.Println(err)
-		v.Options.Theme = DefaultTheme
-		return
+	if strings.HasPrefix(strings.TrimSpace(c.Args), "{") {
+		if err := json.Unmarshal([]byte(c.Args), &v.Options.Theme); err != nil {
+			fmt.Println("invalid Set Theme JSON:", err)
+			v.Options.Theme = DefaultTheme
+			return
+		}
+	} else {
+		if theme, ok := Themes()[c.Args]; ok {
+			v.Options.Theme = theme
+		} else {
+			v.Options.Theme = DefaultTheme
+			return
+		}
```

Comment:
Let's check if the theme is in the `Themes` map first (it won't be if given JSON) and then try parsing JSON and then return if error. I think that should clean up the error handling and if statements a bit